### DISCUSSION TITLE
Persist SQL Console between sessions?

### DISCUSSION THREAD

**Discussion by pkirkaas** [dbeaver/dbeaver#12020] [Q&A] [answered]
I often create multiple one-line queries to examine table data in the SQL console. I don't need to persist them in separate SQL files, but when I close DBeaver & re-open it, my other tabs (tables, etc) are present, but the SQL console is gone, and I have to re-enter my simple, standard queries.

Other tools like SQLYog do this.  In the Preferences/Editors/SQL Editor settings, I have selected "Auto-save editor on close", "Auto-save after any modifications", with no result.

Is this possible in DBeaver? If not, that's a feature request.

I am using the latest 21.0.02 version of DBeaver Community

**Comment by uslss** [comment, marked as answer]
@pkirkaas , you can create just one SQL script with your queries, and it will be available in the future (also after app restart).
![image](https://user-images.githubusercontent.com/31996417/113709427-22303e00-96eb-11eb-92a8-9eb32f6c26b9.png)

SQL console was implemented for those queries that will be needed during current session and will not be needed by the user in the future, so it cannot be saved

**Comment by pkirkaas** [comment]
Thank you, just what I wanted!
